makefile

```diff
@@ -0,0 +1,30 @@
+-include .env
+
+# Default command to run when no target is specified
+all: test-local
+
+# Make anvil remote
+anvil-remote:
+	@echo "Starting Anvil with remote chain..."
+	# Start Anvil with the provided rpc url chain and capture its PID
+	anvil -f $(TEST_RPC_URL) & echo $$! > anvil_pid.txt
+
+# Make anvil local
+anvil-local:
+	@echo "Starting Anvil with local fork..."
+	# Start Anvil with the provided rpc url chain and capture its PID
+	anvil -f $(TEST_RPC_URL) & echo $$! > anvil_pid.txt
```

Comment:
This shouldn't fork the RPC right? It's currently the same as `anvil-remote`. Also I don't know about sending anvil into the background. It's convenient, but if we decide to keep it, then we should have a command that kills the anvil instance for the user by reading the file.

Reply:
No worries! created an issue for it https://github.com/ScopeLift/stealth-address-sdk/issues/38